apps/condo/domains/notification/tasks/deliverMessage.js

```diff
@@ -226,5 +226,5 @@ async function deliverMessage (messageId) {
 }
 
 module.exports = {
-    deliverMessage: createTask('deliverMessage', deliverMessage, { priority: 1 }),
+    deliverMessage: createTask('deliverMessage', deliverMessage, {}, 'high'),
```

Comment:
It's better to user medium for all messages, since it's default behaviour. And tweet some delays instead. Few delays to deliver important staff (like sms codes), and few delays to lower status (like message batch)

Reply:
In fact, I don’t think this is possible without significant changes in the logic of how messages are sent